config/locales/in_person_proofing/en.yml

```diff
@@ -134,11 +143,11 @@ en:
       address: Enter your current residential address
       barcode: Show this barcode and your state‑issued ID at a Post Office to finish
         verifying your identity
-      cta: Try verifying your ID in person
+      cta: Try verifying your identity at a Post Office
       id_address: Address on your ID
       po_search:
         location: Find a participating Post Office
-      prepare: Verify your identity in person
+      prepare: Verify your identity at a Post Office
       state_id_milestone_2: Enter the information on your state‑issued ID
```

Comment:
There are 2 instance of `in_person_proofing.headings.prepare` found in the repo that are not in specs. The first is the title on the page- the other instance is changing the tab title. See screenshot. 

<img width="1512" alt="Screenshot 2023-11-02 at 10 15 55 AM" src="https://github.com/18F/identity-idp/assets/125507397/80f1cfc3-49c8-498f-8325-adb4b20d4a16">